src/build.py

```diff
@@ -1385,14 +1385,21 @@ def CompileLLVMTortureBinaryen(name, em_config, outdir, fails, opt, lld):
     config = 'binaryen-lld'
   else:
     config = 'binaryen'
-  unexpected_result_count = compile_torture_tests.run(
-      cc=cc, cxx=cxx, testsuite=GCC_TEST_DIR,
-      sysroot_dir=INSTALL_SYSROOT,
-      fails=fails,
-      exclusions=LLVM_TORTURE_EXCLUSIONS,
-      out=outdir,
-      config=config,
-      opt=opt)
+  try:
+    if not lld
+      os.environ['EMCC_EXPERIMENTAL_USE_LLD'] = '0'
+    unexpected_result_count = compile_torture_tests.run(
+        cc=cc, cxx=cxx, testsuite=GCC_TEST_DIR,
+        sysroot_dir=INSTALL_SYSROOT,
+        fails=fails,
+        exclusions=LLVM_TORTURE_EXCLUSIONS,
+        out=outdir,
+        config=config,
+        opt=opt)
+  finally:
+    if not lld:
+      del os.environ['EMCC_EXPERIMENTAL_USE_LLD']
```

Comment:
Not a fan of modifying os.environ in general
And in theory this can cause weirdness because it doesn't perfectly clean up (if the var was set before, now it's unset)
But in practice we control all the paths through here
And we'll remove the flag in a week or two tops, so this is as clean as it needs to be

Reply:
Yeah, I figure the same.  Plus I'm only moving existing code